### ISSUE TITLE
Share to Ente won't complete

### ISSUE THREAD

**Issue by s38b35M5** [ente/ente#1214]
### Description

When viewing non-ente-uploaded photos in my non-Ente gallery app, or my file browser, I want to upload them to Ente, but the share process is incomplete or bugged. I don't want Ente App to watch these folders and upload all downloaded pictures; I prefer a way to selectively upload. (This could possibly spin off into a FR for adding photos from the app instead of simply watching folders)

To reproduce:

- Open file browser app of choice or gallery app (I use FOSS Gallery and Google Files)
- Select photo(s) and choose the share button
- Choose Ente

Result: Ente opens, but does not open the album/destination selection screen as expected

Desired/Expected result: Ente should open the album picker screen to allow user to select where uploaded photos will go and then upload the selected file(s).

Side note: Why do I have to choose an album for upload? Why can't photos just be uploaded individually or in groups to the root photo folder/album?

### Version

v0.8.70

### What product are you using?

Ente Photos

### What platform are you using?

Mobile - Android

**Comment by vishnukvmd** [comment]
Will try to reproduce this.

In the meanwhile, could you please pick such items from the "On device" section of the app (w/ second tab), and click on the Upload icon on the top right corner? It'll serve as a temporary fix while we figure out what's going wrong with the share-action.

**Comment by s38b35M5** [comment]
Ah, thank you for that tip. I didn't know.

**Comment by vishnukvmd** [comment]
Hey, we're unfortunately not able to reproduce this on our test devices. It's working as expected, where Ente starts up with an option to pick an album.

Could you please share details about the device where you're experiencing this issue?

**Comment by s38b35M5** [comment]
Still repeatable for me.

Pixel 5
LineageOS 21 (Android 14)
Share attempted from Google Files app (com.android.documentsui v14)

**Comment by s38b35M5** [comment]
> Will try to reproduce this.
> 
> In the meanwhile, could you please pick such items from the "On device" section of the app (w/ second tab), and click on the Upload icon on the top right corner? It'll serve as a temporary fix while we figure out what's going wrong with the share-action.

Just a reminder that the location of this feature is not obvious or intuitive. I had tk come back to this comment to remember how to upload singles when the share feature (still broken for me) isn't ideal.

**Comment by vishnukvmd** [comment]
> location of this feature is not obvious or intuitive

Thanks for the feedback, if you have ideas on a more discoverable place to keep this button, do let us know!

**Comment by s38b35M5** [comment]
> > location of this feature is not obvious or intuitive
> 
> Thanks for the feedback, if you have ideas on a more discoverable place to keep this button, do let us know!

Certainly! I feel that locating it in the same area as when you click the upload button in the top right --which currently only shows folders to backup -- woukd allow simpler uploading of individual files as needed.

**Comment by AmanRajSinghMourya** [comment]
The Android sharing flow has changed substantially since Photos 0.8.70, including how new share intents are delivered and handled. Could you please retest this with the latest Photos release? If it still occurs, please contact support at support@ente.com

**Comment by s38b35M5** [comment]
> The Android sharing flow has changed substantially since Photos 0.8.70, including how new share intents are delivered and handled. Could you please retest this with the latest Photos release? If it still occurs, please contact support at [support@ente.com](mailto:support@ente.com)

Confirmed no longer an issue
